pyproject.toml

```diff
@@ -58,6 +58,8 @@ include = ["/src/safe_cli"]
 
 [tool.hatch.build.targets.wheel]
 packages = ["/src/safe_cli"]
+[tool.hatch.metadata]
+allow-direct-references = true
```

Comment:
Just to be able to install direct dependencies from github.
This must be removed after safe-eth-py release to the correct version.